airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -181,13 +182,15 @@ def _construct_volume(name, claim, host):
         if self.kube_config.logs_volume_subpath:
             volume_mounts[self.logs_volume_name]['subPath'] = self.kube_config.logs_volume_subpath
 
-        if self.kube_config.dags_in_image or self.kube_config.dags_configmap:
+        if (self.kube_config.dags_in_image or
+                self.kube_config.dags_configmap or
+                self.kube_config.lib_configmap):
```

Comment:
Remove line

Reply:
do we need to add a section like this for lib?